Anytype/Sources/ServiceLayer/Object/TypesService/TypesService.swift

```diff
@@ -32,6 +33,17 @@ final class TypesService: TypesServiceProtocol {
         return ObjectType(details: objectDetails)
     }
     
+    func deleteType(_ type: ObjectType, spaceId: String) async throws {
+        guard !type.readonly else {
```

Comment:
Is it really needed? Middleware should check it. Also, i think, in feature we can create ObjectService with common object methods like `workspaceObjectListRemove`

Reply:
You make a limitation. You can delete only ObjectType. If view layer have only id, i can't delete object.